packages/components/src/tabs/tab.tsx

```diff
@@ -22,20 +27,35 @@ export const Tab = forwardRef<
 	HTMLButtonElement,
 	Omit< WordPressComponentProps< TabProps, 'button', false >, 'id' >
 >( function Tab( { children, tabId, disabled, render, ...otherProps }, ref ) {
-	const context = useTabsContext();
-	if ( ! context ) {
+	const { store, instanceId } = useTabsContext() ?? {};
+
+	// If the active item is not connected, the tablist may end up in a state
+	// where none of the tabs are tabbable. In this case, we force all tabs to
+	// be tabbable, so that as soon as an item received focus, it becomes active
+	// and Tablist goes back to working as expected.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

Comment:
We might want to refactor those early returns in the future. Do we have an open issue for that?

Reply:
Do you mean the "Subcomponent must be wrapped in the top level component" warnings?  Just double-checking before I go ahead and create the issue.

Reply:
Yes 👍

Reply:
https://github.com/WordPress/gutenberg/issues/66530